Digital_Fusion/src/components/Card/Card.jsx

```diff
@@ -0,0 +1,10 @@
+import "../Card/Card.css";
+export function Card({ card, openPopup }) {
+  return (
+    <div className="card" key={card.id} onClick={() => openPopup(card)}>
```

Comment:
`key={card.id} ` is not needed here

The `key` attribute is needed only when you render an array of components. You shouldn’t put it inside any component initially